src/Dropdown.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect } from "react";
+import cn from "classnames";
+import { Person } from "./types/Person";
+
+type Props = {
+  onSelect: (person: Person) => void;
+  people: Person[];
+};
+
+export const Dropdown: React.FC<Props> = ({ onSelect, people }) => {
+  const [query, setQuery] = useState("");
+  const [debouncedQuery, setDebouncedQuery] = useState("");
+  const [isTyping, setIsTyping] = useState(false);
+
+  useEffect(() => {
+    const timerId = setTimeout(() => {
+      setDebouncedQuery(query);
+      setIsTyping(false);
+    }, 1000);
```

Comment:
![image](https://user-images.githubusercontent.com/122989585/219865071-2b65ae18-7d8d-4054-86e3-09fa8b6cea56.png)
now you have hardcoded time value like 1000